rllib/core/models/catalog.py

```diff
@@ -367,7 +367,7 @@ def get_encoder_config(
                 encoder_config = CNNEncoderConfig(
                     input_dims=observation_space.shape,
                     cnn_filter_specifiers=model_config_dict["conv_filters"],
-                    cnn_activation=activation,
+                    cnn_activation=model_config_dict["conv_activation"],
```

Comment:
This was a bug: Used the `fcnet_activation` (tanh by default), instead of conv one (relu by default).
This bug completely broke our Pong benchmark.

Reply:
FYI @ArturNiederfahrenhorst

Reply:
It's all good man! :D